modules/pubmaticBidAdapter.js

```diff
@@ -624,7 +624,7 @@ function _handleTTDId(eids, validBidRequests) {
  * Produces external userid object in ortb 3.0 model.
  */
 function _addExternalUserId(eids, value, source, atype) {
-  if (utils.isStr(value)) {
+  if (utils.isStr(value) || (source === 'parrable.com' && utils.isPlainObject(value) && !utils.isEmpty(value))) {
```

Comment:
In the function below this one (`_handleEids`) there are two special case functions: `_handleDigitrustId` and `_handleTTDId`.
I would recommend that we create our own `_handleParrableId` instead of polluting `_addExternalUserId` with our own logic.

Reply:
makes sense. making the change.